test/lit/passes/inlining_enable-tail-call.wast

```diff
@@ -884,56 +794,6 @@
  )
 )
 
-(module
- ;; Same as above, but this time the child is not unreachable.
```

Comment:
Why remove this test?

Reply:
I think I somehow missed them while rebasing.